apps/settings/elements/addon_details.html

```diff
@@ -0,0 +1,49 @@
+<element name="addon-details" extends="section">
+  <template>
+
+    <gaia-header action="back" data-href="#addons">
+      <h1 id="addon-details-header" data-l10n-id="addon-details-header"></h1>
+    </gaia-header>
+
+    <div id="addon-details-body">
```

Comment:
We usually use class names to avoid possible conflicts because there are many panels in settings app.